src/analysis/retail/evoker/preservation/modules/features/Checklist/Component.tsx

```diff
@@ -0,0 +1,60 @@
+import SPELLS from 'common/SPELLS';
+import { TALENTS_EVOKER } from 'common/TALENTS';
+import { SpellLink } from 'interface';
+import PreparationRule from 'parser/retail/modules/features/Checklist/PreparationRule';
+import Checklist from 'parser/shared/modules/features/Checklist';
+import {
+  AbilityRequirementProps,
+  ChecklistProps,
+} from 'parser/shared/modules/features/Checklist/ChecklistTypes';
+import GenericCastEfficiencyRequirement from 'parser/shared/modules/features/Checklist/GenericCastEfficiencyRequirement';
+import Requirement from 'parser/shared/modules/features/Checklist/Requirement';
+import Rule from 'parser/shared/modules/features/Checklist/Rule';
+
+const PreservationEvokerChecklist = ({ combatant, castEfficiency, thresholds }: ChecklistProps) => {
+  const AbilityRequirement = (props: AbilityRequirementProps) => (
+    <GenericCastEfficiencyRequirement
+      castEfficiency={castEfficiency.getCastEfficiencyForSpellId(props.spell)}
+      {...props}
+    />
+  );
+  return (
+    <Checklist>
+      <Rule
+        name="Don't overcap Essence"
+        description={
+          <>
+            Abilities that cost essence like <SpellLink id={SPELLS.EMERALD_BLOSSOM.id} /> are very
+            powerful and are a core part of your healing as a Preservation Evoker. Make sure you are
+            not overcapping on Essence too often.
+          </>
+        }
+      >
+        <Requirement name={<>Essence Wasted</>} thresholds={thresholds.essenceDetails} />
+      </Rule>
+      <Rule
+        name="Use your empowered spells wisely"
+        description={
+          <>
+            Empowered spells like <SpellLink id={TALENTS_EVOKER.DREAM_BREATH_TALENT.id} /> make up a
+            large portion of your healing. Be sure to use them often while keeping your overheal
+            low.
+          </>
+        }
+      >
+        <AbilityRequirement spell={SPELLS.DREAM_BREATH_CAST.id} />
+        <Requirement
```

Comment:
I recommend wrapping this in a `hasTalent` check since this is a talent.

Reply:
It is impossible to not select Dream Breath so it will always be active. Though I suppose there might be someone out there who kills a boss with no talents. Will fix